.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +1,8 @@
-<!-- Thank you for sending a PR! -->
-<!-- Relevant links - https://github.com/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message -->
-<!-- https://github.com/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines -->
-<!-- Please perform the following checks and mark all the boxes accordingly. -->
-<!-- You can remove the checklist items that don't apply to your PR. -->
+<!--
+Thank you for contributing!
+Please fill in the following checklist, removing items that do not apply.
+See also https://github.com/tldr-pages/tldr/blob/main/CONTRIBUTING.md
```

Comment:
Mentioning specific sections of the contributing guide helps take people to the right places faster.

Reply:
Hmmm, I didn't notice that. Good spot.

Reply:
I did when adding it, the goal was for the user to potential see it, so that if they missed the top then they still had the checklist and then the other way round with not seeing/ignoring the checklist.

Reply:
@CleanMachine1 so now we have the general link in the comment and the specific links in the body, that seems to still capture your intention, no?

Mind that you cannot click a link in the comment and it is not highlighted in the editor, that is why I deem it more useful this way around.